composer.json

```diff
@@ -26,8 +27,7 @@
   },
   "suggest": {
     "spryker/console": "*",
-    "spryker/silex": "^2.0.0",
-    "spryker/symfony": "^3.0.0"
+    "spryker/silex": "^2.0.0"
```

Comment:
This shouldnt be version numbers but reasons why (text).

Reply:
I know, but why it is merged by you before I have the chance to fix that?

Reply:
you can always make a follow up.